src/Commands/BuildToolsCommand.php

```diff
@@ -658,9 +659,13 @@ protected function expandSourceAliases($source)
      */
     protected function autodetectUpstream($siteDir)
     {
-        return 'Empty Upstream';
-        // or 'Drupal 7' or 'WordPress'
-        // return 'Drupal 8';
+        if (file_exists($siteDir . '/web/wp-config.php')) {
```

Comment:
A variant of this line was already committed as part of a separate PR. We could check for both `web/wp-config.php` and `web/wp/wp-config.php`, or test some file that is always in the docroot, even when core is relocated, if such a candidate exists.